features/shared/checkout_previous_branch_after_success/same_current_branch/previous_branch_same.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Allow checking out the correct previous Git branch after running a Git Town commmand that leaves the current and previous branches intact
+
+  As a developer running `git checkout -` after running a Git Town command
+  I want to end up on the expected previous branch
+  So that I can consistently and effectively use Git's commands
```

Comment:
So that Git Town does not interfere with my productive use of the Git command history